e2e/pod.go

```diff
@@ -105,17 +105,49 @@ func waitForDeploymentComplete(name, ns string, c kubernetes.Interface, t int) e
 	return nil
 }
 
-func getCommandInPodOpts(f *framework.Framework, c, ns string, opt *metav1.ListOptions) (framework.ExecOptions, error) {
+func findContainerName(f *framework.Framework, c, ns, cn string, opt *metav1.ListOptions) (string, string, error) {
+	pn := ""
+	podList, err := f.PodClientNS(ns).List(context.TODO(), *opt)
+	framework.ExpectNoError(err)
+	if len(podList.Items) == 0 {
+		return "", "", errors.New("podlist is empty")
+	}
+	found := false
+	if cn != "" {
+		for i := range podList.Items {
+			for j := range podList.Items[i].Spec.Containers {
+				if podList.Items[i].Spec.Containers[j].Name == cn {
+					pn = podList.Items[i].Name
+					found = true
+					break
```

Comment:
if found  return from here? so that we can avoid the new variable `found` and also an if check?